examples/simple/igraph_ring.c

```diff
@@ -42,5 +42,14 @@ int main() {
     /* Destroy the graph. */
     igraph_destroy(&graph);
 
+    igraph_ring(&graph,1,IGRAPH_UNDIRECTED,0,1);
```

Comment:
You've changed the example, so the output should change as well, you can fix this in the igraph_ring.out file.

Reply:
Can you explain what should i change here?

Reply:
Look at the output of this example, check if it's correct, and if so, put it in  examples/simple/igraph_ring.out so the tests pass.